### ISSUE TITLE
Migrate to Gatsby5

### ISSUE THREAD

**Issue by bellons91** [hikerpig/gatsby-project-kb#74]
Gatsby 5 has been released in late 2022.
I think it can be useful to migrate the theme (and the related demo) to Gatsby5.

https://www.gatsbyjs.com/docs/reference/release-notes/v5.13/

**Comment by hikerpig** [comment]
Yes indeed, last year I've been very busy so did not have time to look at it.
I will try to do this in Q1.
